### PR TITLE
create PromotionText.jsx and PromotionTextjson.json

### PR DESCRIPTION
**Pull request by amirhosein0980**
create PromotionText.jsx and PromotionTextjson.json
Added PromotionText component to ui
Please check the files to merge

### CODE REVIEW

File: src/components/widgets/PromotionTextjson.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
```
Comment (chalist) [inline src/components/widgets/PromotionTextjson.json:1]: remove `json` from file name. `PromotionText.json` will be enough.
and move file to folder `src/data`

File: src/components/widgets/PromotionText.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import DataPromotionText from '@/components/widgets/PromotionTextjson.json';
+
+function PromotionText() {
+  return (
+    <>
+      <div className="w-4/5 h-80 m-auto mt-2.5">
+        {DataPromotionText.map((item) => (
+          <div className="w-full h-3/6 rounded-br-2xl rounded-bl-2xl rounded-tr-2xl rounded-tl-2xl" style={{ backgroundColor: 'rgb(229,245,247)' }} >
+            <div className="w-full h-3/6 flex justify-center items-center font-semibold" style={{ color: 'rgb(56,121,125)' }}>{item.data.icon}</div>
+            <div className="w-full h-3/6 flex justify-center items-center font-semibold" style={{ color: 'rgb(56,121,125)', direction: 'rtl' }}>{item.data.content}</div>
```
Comment (chalist) [inline src/components/widgets/PromotionText.jsx:10]: remove all style attrs. Just use `tailwindcss` classes.
for rtl, you can use a custom class in `assets/css` folder.

Comment (chalist) [reply src/components/widgets/PromotionText.jsx:10]: Also you need to use `img` for icon!

File: src/components/widgets/PromotionTextjson.json (hunk 1 above)
Comment (amirhosein0980) [reply src/components/widgets/PromotionTextjson.json]: Edited

File: src/components/widgets/PromotionText.jsx (hunk 2 above)
Comment (amirhosein0980) [reply src/components/widgets/PromotionText.jsx]: Edited

**Pull request merged**